CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## AMI Release v20210501
+
+* amazon-eks-gpu-node-1.19-v20210501
+* amazon-eks-gpu-node-1.18-v20210501
+* amazon-eks-gpu-node-1.17-v20210501
+* amazon-eks-gpu-node-1.16-v20210501
+* amazon-eks-gpu-node-1.15-v20210501
+* amazon-eks-arm64-node-1.19-v20210501
+* amazon-eks-arm64-node-1.18-v20210501
+* amazon-eks-arm64-node-1.17-v20210501
+* amazon-eks-arm64-node-1.16-v20210501
+* amazon-eks-arm64-node-1.15-v20210501
+* amazon-eks-node-1.19-v20210501
+* amazon-eks-node-1.18-v20210501
+* amazon-eks-node-1.17-v20210501
+* amazon-eks-node-1.16-v20210501
+* amazon-eks-node-1.15-v20210501
+
+Binaries used to build these AMIs are published:
+
+s3://amazon-eks/1.19.6/2021-01-05/
+s3://amazon-eks/1.18.9/2020-11-02/
+s3://amazon-eks/1.17.12/2020-11-02/
+s3://amazon-eks/1.16.15/2020-11-02/
+s3://amazon-eks/1.15.12/2020-11-02/
+
+Notable changes:
+
+* Kernel patches for Kubernetes v1.18 and below (CVE ALAS2-2021-1627)
```

Comment:
We should reference the 4.14 kernel here. I was confused by the kubernetes reference.